src/services/SendToGovNotifyService.ts

```diff
@@ -0,0 +1,613 @@
+		
+  		const pdfParams = {
+  			Bucket: bucket,
+  			Key: key,
+  		};
+
+  		this.logger.info("Fetching Yoti PDF from bucket");
+  	try {
+  		const yotiPdf = await s3Client.send(new GetObjectCommand(pdfParams));
+		  if (yotiPdf.Body instanceof Readable) {
+  			const body = await streamToBuffer(yotiPdf.Body);
+  			const encoded = body.toString("base64");
+  			return encoded;
+		  }
+  	} catch (error) {
+  		this.logger.error({ message: "Error fetching Yoti PDF from S3 bucket", error });
+  	}	
+  }
+
+  async fetchMergedPdf(sessionId: string, yotiSessionId: string): Promise<any> {
+  	const f2fSessionInfo = await this.f2fService.getSessionById(sessionId);
+  		if (!f2fSessionInfo) {
+  			this.logger.warn("Missing details in SESSION table", {
+  				messageCode: MessageCodes.SESSION_NOT_FOUND,
+  			});
+  			throw new AppError(HttpCodesEnum.BAD_REQUEST, "Missing details in SESSION table");
+  		}
+  		
+
+  	const streamToBuffer = async (stream: Readable): Promise<Buffer> => {
+  		return new Promise((resolve, reject) => {
+  			const chunks: Uint8Array[] = [];
+  			stream.on("data", (chunk) => chunks.push(chunk));
+  			stream.on("end", () => resolve(Buffer.concat(chunks)));
+  			stream.on("error", reject);
+  		});
+  	};
+	 
+  		const s3Client = new S3Client({
+  			region: process.env.REGION,
+  			maxAttempts: 2,
+  			requestHandler: new NodeHttpHandler({
+  				connectionTimeout: 29000,
+  				socketTimeout: 29000,
+  			}),
+  		});
+
+  	const bucket = this.environmentVariables.yotiLetterBucket();
+  	const folder = this.environmentVariables.mergedPdfBucketFolder();
+  	const key = `${folder}/${yotiSessionId}`;
+		
+  	const pdfParams = {
+  		Bucket: bucket,
+  		Key: key,
+  	};
+	  
+  		this.logger.info("Fetching merged PDF from bucket");
+
+  	try {
+  		const mergedPdf = await s3Client.send(new GetObjectCommand(pdfParams));
+		  if (mergedPdf.Body instanceof Readable) {
+  			const body = await streamToBuffer(mergedPdf.Body);
+  			const encoded = body.toString("base64");
+  			return encoded;
+		  }
+  	} catch (error) {
+  		this.logger.error({ message: "Error fetching merged PDF from S3 bucket", error });
+  	}	
+  }
+
+
+  async sendReminderEmail(message: ReminderEmail): Promise<EmailResponse> {
+  	this.logger.info("Sending reminder email");
+
+  	try {
+  		const options = {
+  			reference: message.referenceId,
+  		};
+
+  		const emailResponse = await this.sendGovNotificationEmail(
+  			this.environmentVariables.getReminderEmailTemplateId(this.logger),
+  			message,
+  			options,
+  			this.environmentVariables.reminderEmailsGovNotifyUrl(),
+  		);
+  		return emailResponse;
+  	} catch (err: any) {
+  		this.logger.error("Failed to send Reminder Email", {
+  			messageCode: MessageCodes.FAILED_TO_SEND_REMINDER_EMAIL,
+  		});
+  		throw new AppError(
+  			HttpCodesEnum.SERVER_ERROR,
+  			"sendReminderEmail - Failed sending Reminder Email",
+  		);
+  	}
+  }
+
+  async sendDynamicReminderEmail(
```

Comment:
Why is this service having all the reminderEmail related functions. are we extracting this as a util file which will be used by the existing Govnotify code to send the reminder emails?